dash_player/DashPlayer.py

```diff
@@ -5,94 +5,161 @@
 
 class DashPlayer(Component):
     """A DashPlayer component.
-A Dash component for playing a variety of URLs, including file paths,
-YouTube, Facebook, Twitch, SoundCloud, Streamable, Vimeo, Wistia, Mixcloud,
-and DailyMotion.
+    A Dash component for playing a variety of URLs, including file paths,
```

Comment:
Unless you want to include `black` in the build process (which you can do if you like, we do it in the core but not in most other components), don't run it on build artifacts like this. I can't get the `--exclude` and `--include` args to work right, but something like:
```
black dash_player/__init__.py tests *.py
```
(and again, put `black` and `black --check` both in package scripts to make it easy to run locally)

Reply:
Ok, I'll leave `black` out of the CI process for now